tests/utils/Constants.sol

```diff
@@ -10,11 +10,13 @@ abstract contract Constants {
     uint128 public constant CLAIM_AMOUNT = 10_000e18;
     uint128 public constant CLIFF_AMOUNT = (CLAIM_AMOUNT * CLIFF_DURATION) / TOTAL_DURATION;
     UD2x18 public immutable CLIFF_PERCENTAGE = (ud(CLIFF_AMOUNT).div(ud(CLAIM_AMOUNT)).intoUD2x18());
-    uint256 public constant MAX_FEE = 100e8; // $100
-    uint256 public constant MINIMUM_FEE = 3e8; // $3 fee
-    uint256 public constant MINIMUM_FEE_IN_WEI = (1e18 * MINIMUM_FEE) / 3000e8; // at $3000 per ETH price
+    uint256 public constant MAX_FEE_USD = 100e8; // $100
+    uint256 public constant MIN_FEE_USD = 3e8; // $3 fee
+    uint256 public constant MINIMUM_FEE_IN_WEI = (1e18 * MIN_FEE_USD) / 3000e8; // at $3000 per ETH price
```

Comment:
Similar to how we are naming fee in USD, we can just say `MIN_FEE_WEI`

Reply:
Yeah